pkg/horoscope/data.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package horoscope
+
+import (
+	"fmt"
+	"github.com/aclements/go-moremath/stats"
+	"github.com/pingcap/parser/ast"
+	"golang.org/x/perf/benchstat"
+	"math"
+
+	"github.com/chaos-mesh/horoscope/pkg/executor"
+)
```

Comment:
Should group imports.

Reply:
Updated on https://github.com/chaos-mesh/horoscope/pull/9/commits/081ff21a863b0c2ca0faccb718aaba9beb87dc1c